src/clint.py

```diff
@@ -3596,6 +3604,7 @@ def ParseArguments(args):
     _SetCountingStyle(counting_style)
     _SuppressErrorsFrom(suppress_errors_file)
     _RecordErrorsTo(record_errors_file)
+    _cpplint_state.stdin_filename = stdin_filename
```

Comment:
I did not feel like adding (two) methods just for that (following the existing pattern here), but I can change it if necessary.